docs/src/pages/components/alert/examples/AlertHeadingExample.tsx

```diff
@@ -0,0 +1,12 @@
+import { Alert } from '@aws-amplify/ui-react';
```

Comment:
Adding all examples in an examples directory to keep the code shown to uses and code run in sync.